draft-irtf-cfrg-opaque.md

```diff
@@ -1005,13 +1014,13 @@ IKM = epkU^eskS || epkU^skS || pkU^eskS
 #### OPAQUE-3DH Encryption and Key Confirmation {#3dh-core}
 
 Clients and servers use keys Km2 and Km3 in computing KE2.mac and KE3.mac,
-respectively. These values are computed as HMAC(mac_key, transcript), where
-mac_key and transcript are as follows:
+respectively. These values are computed as HMAC(mac_key, Hash(info || transcript)),
```

Comment:
`suffix` would be a fine name for this, I think?

Reply:
sounds good!

Reply:
Agreed with not calling this `transcript`. Also, maybe the clarity of this could be improved, by explicitly writing out the computation of KE2.mac and KE2.mac in an equation form (similarly to how the `auth_tag` computation is written for the envelope):

- `Ke2.mac = HMAC(Km2, concat(info, KE2.enc_server_info))`
- `Ke3.mac = HMAC(Km3, concat(info, KE2.enc_server_info, KE2.mac))`

That way, you don't even need to introduce the `transcript` or `suffix` variable :)